code/modules/mob/living/carbon/human/human_organs.dm

```diff
@@ -169,3 +169,24 @@ I use this to standardize shadowling dethrall code
 	for(var/obj/item/organ/external/limb in bodyparts)
 		if(limb.status & ORGAN_SPLINTED)
 			splinted_limbs += limb
+
+/mob/living/carbon/human/proc/handle_severe_burns()
+	var/burned_limbs = 0 // amount of burn wounds that aren't salved a person has
+	for(var/obj/item/organ/external/O in bodyparts)
```

Comment:
```suggestion
	for(var/obj/item/organ/external/O as anything in bodyparts)
```

Reply:
Will get this done when i get round to fixing the conflicts.